### ISSUE TITLE
Feature Request: Buttons

### ISSUE THREAD

**Issue by BestLeaks** [Glitchii/embedbuilder#25]
Could you please add support for buttons?

**Comment by Glitchii** [comment]
How would this work? What would be the purpose of buttons?

**Comment by Glitchii** [comment]
TLDR; I will not be adding the buttons as it goes against the main purpose of the embed builder.

I know what buttons are. I just don't really see the purpose for them.
The main goal of the embed builder is to be a simple and light-weight embed visualizer that can be embedded in websites for Discord bots. If I'm adding buttons then I'd also have to add all other components. It's not supposed to be a full Discord visualizer with buttons or any other components etc. Just a simple **embed builder** that also supports messages.

You're probably looking for something like this https://autocode.com/tools/discord/embed-builder/, whoever I don't think they even allow embedding source-code into your site, neither do I think it's opensource.
